connectors/kafka/src/main/java/forklift/controller/KafkaController.java

```diff
@@ -184,6 +141,10 @@ private void controlLoop() {
             throw t;
         } finally {
             running = false;
+            synchronized (this) {
+                //wake up any threads waiting on a control loop operation
+                this.notifyAll();
```

Comment:
Good catch, with the removal of the add/remove topic methods this is no longer needed.